src/Mod/Path/PathScripts/post/gcode_pre.py

```diff
@@ -112,7 +112,11 @@ def insert(filename, docname):
 
         # Create a custom and viewobject
         obj = PathCustom.Create("Custom")
-        res = PathOpGui.CommandResources('Custom', PathCustom.Create, PathCustomGui.TaskPanelOpPage, 'Path-Custom', 'Path-Custom', '', '')
+        res = PathOpGui.CommandResources('Custom',
+            PathCustom.Create, PathCustomGui.TaskPanelOpPage,
+            'Path_Custom', 
+            'Path_Custom', '', ''
+            )
```

Comment:
I modified this. Does it look correct or shall I revert?

Reply:
The fourth parameter there is the pixmap, but the fifth is the menu text. I doubt it is supposed to have either an underscore or a dash in it, I'm betting there is some missing translation code -- something like
```suggestion
        res = PathOpGui.CommandResources('Custom',
            PathCustom.Create, PathCustomGui.TaskPanelOpPage,
            'Path_Custom', 
            QtCore.QT_TRANSLATE_NOOP('Path_Custom', 'Custom'), '', ''
            )
```

Reply:
@mlampert care to weigh in?

Reply:
Yes, Dr. Hennes.  That fifth item should include a translation call, like:
`QtCore.QT_TRANSLATE_NOOP("PathCustom", "Custom Path")` as you included in your suggestion.  I believe (but have not tracked down source code function) the first argument in the translate function used is a general location reference, like the module file name, hence "PathCustom", "PathHelix", or "PathPocket."

Reply:
lol, nobody calls me "Dr. Hennes" unless I've just screwed something up and they're about to point it out :). You are right about the first argument to [QT_TRANSLATE_NOOP](https://doc.qt.io/qt-5/i18n-source-translation.html#using-qt-tr-noop-and-qt-translate-noop-in-c): Qt calls it the "Context," it's used to group together various translation items in the .ts file.

Reply:
> `QtCore.QT_TRANSLATE_NOOP("PathCustom", "Custom Path")` as you included in your suggestion. I believe (but have not tracked down source code function) the first argument in the translate function used is a general location reference, like the module file name, hence "PathCustom", "PathHelix", or "PathPocket."

@Russ4262 thanks for bringing this up. I do have a commit that I didn't add to this PR that changes all the translation context fields to have an underscore (for several reasons, because there is a mix in the Path code of camelcase and underscore and also because of the wiki documentation/images have underscores). But I didn't include it because it would create a [kitten eating moster](http://webchick.net/please-stop-eating-baby-kittens) PR. It would also render all/many Path translations obsolete pre-release. So I'm waiting till after 0.19 release to submit it